pkgs/by-name/mo/monado-vulkan-layers/package.nix

```
@@ -0,0 +1,43 @@
+{
+  lib,
+  stdenv,
+  fetchFromGitLab,
+  cmake,
+  unstableGitUpdater,
+  vulkan-headers,
+  vulkan-loader,
+}:
+stdenv.mkDerivation {
+  pname = "monado-vulkan-layers";
+  version = "0-unstable-2024-02-21";
+
+  src = fetchFromGitLab {
+    domain = "gitlab.freedesktop.org";
+    owner = "monado";
+    repo = "utilities/vulkan-layers";
+    rev = "ae43cdcbd25c56e3481bbc8a0ce2bfcebba9f7c2";
+    sha256 = "sha256-QabYVKcenW+LQ+QSjUoQOLOQAVHdjE0YXd+1WsdzNPc=";
+  };
+
```

Nit: might be worth setting

```suggestion

  strictDeps = true;

```